js/interface/map-panel.js

```diff
@@ -31,6 +74,41 @@ Fliplet.InteractiveMap.component('map-panel', {
       }
     },
     openMapPicker() {
+      this.dataSourceId = __widgetData[this.appId].data.markersDataSourceId
```

Comment:
What? 🤔 

1. `__widgetData` is private and should not be used directly
2. keys in `__widgetData` refer to the Widget Instance ID, not an App ID
3. you want to use `Fliplet.Widget.getData(). markersDataSourceId` as per Fliplet documentation (this only works in widget interfaces)

Reply:
Fixed